### ISSUE TITLE
CODEGEN requires a MATLAB Coder license.

### ISSUE THREAD

**Issue by BJCaasenbrood** [mathworks/ci-configuration-examples#22]

**Comment by acampbel** [comment]
Hi @BJCaasenbrood, 

Thanks for looking into our CircleCI orb! 

While most products are there, not all products are licensed, even for public repositories. In particular what we call transformation products (like Coders and Compilers) are not available with the free licensing that comes along with public repositories. 

If you already have a dedicated license for your CI builds, you can possibly use that for your CircleCI jobs (may require Client Access Licenses) and these can remain public or they can be applied to private repos as well or in addition. Is this something you would be interested in? If so, send an email to continuous-integration@mathworks.com along with the output of the `ver` command in MATLAB (this helps us find your account team).

Thanks!
Andy

**Comment by BJCaasenbrood** [comment]
He Andy, I'll look into it, thanks for quick reply!

**Comment by acampbel** [comment]
@BJCaasenbrood are you working with an academic license? If so it may be easier for you to leverage this approach. Definitely encourage you to send us an email and we can help with getting this going.

**Comment by BJCaasenbrood** [comment]
@acampbel Yes, I do possess an academic MATLAB license (currently). I've reached out to [continuous-integration@mathworks.com](mailto:continuous-integration@mathworks.com) regarding the inquiry. I'll keep you up-to-date.
